root/statistics/Layout.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2018 Shamroy Pellew
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const Layout = require('../layout');
+const Tabs = require('../components/Tabs');
+const {l} = require('../static/scripts/common/i18n');
+
+const LinkStatisticsTab = (link, title) => (
+  <span className="mp"><a href={link}>{title}</a></span>
+);
+
+const infoLinks = [['index', LinkStatisticsTab('/statistics', l('Overview'))], ['countries', LinkStatisticsTab('/statistics/countries', l('Countries'))], ['languages-scripts', LinkStatisticsTab('/statistics/languages-scripts', l('Languages/Scripts'))], ['coverart', LinkStatisticsTab('/statistics/coverart', l('Cover Art'))], ['relationships', LinkStatisticsTab('/statistics/relationships', l('Relationships'))], ['edits', LinkStatisticsTab('/statistics/edits', l('Edits'))], ['formats', LinkStatisticsTab('/statistics/formats', l('Formats'))], ['editors', LinkStatisticsTab('/statistics/editors', l('Editors'))], ['timeline', LinkStatisticsTab('/statistics/timeline/main', l('Timeline'))]];
+
+const StatisticsLayout = ({title, fullWidth, page, children}) => {
+  const htmlTitle = l('Database Statistics - {title}', {title: title});
+  return fullWidth ? (
+    <Layout fullWidth title={htmlTitle}>
+      {/* [%- PROCESS "statistics/macros-header.tt" -%] */}
+      <div className="statisticsheader">
+        <h1>{l('Database Statistics')}</h1>
+      </div>
+      <Tabs list={infoLinks} sel={page} />
```

Comment:
Should be reworked to use the component `Tabs` defined in https://github.com/metabrainz/musicbrainz-server/commit/4149827edbb414f35342e7d724d382c1e3e9746d instead. See comment to https://github.com/metabrainz/musicbrainz-server/pull/609/commits/587fee5a5fb5e8c7dc52368b281ecac856db1123.